csharp/extractor/Semmle.Util/ToolStatusPage.cs

```diff
@@ -0,0 +1,236 @@
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace Semmle.Util
+{
+    /// <summary>
+    /// Represents diagnostic messages for the tool status page.
+    /// </summary>
+    public class DiagnosticMessage
```

Comment:
Why are these classes placed in the same file? (I was surprised to find `DiagnosticMessage` in `ToolStatusPage.cs`.) Should `ToolStatusPage.cs` become a folder instead?